vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -228,7 +161,7 @@ standard images that can be used, but generally we expect our customers to provi
 Deployment packages three images into a single Pod: the Entando App Image in question, AppBuilder and Component Manager.
```

Comment:
super awkward. "deployment packages" reads like "packages" is a noun. "A deployment packages"? to set the scene for a verb?

would remove "in question"; that reads like the entando app image is under interrogation or something. no reason to call it out with a qualifier.

Reply:
don't know why Deployment is capitalized, it is "An Entandoapp deployment ..."

changed second part to " the EntandoApp image discussed here, AppBuilder..."

Reply:
ignore my first comment! i'm sure you were confused... i had missed that the sentence didn't start with "Deployment"; i was commenting on "Deployment packages..." not realizing that this was prefaced by "An EntandoApp"